ReleaseStack_WCProductTest test failing

Test `testUpdateProduct()` is failing:

```
Caused by: java.lang.IllegalStateException: Gson().fromJson<JsonElem… JsonElement::class.java) must not be null
        at org.wordpress.android.fluxc.model.WCProductModel.getImages(WCProductModel.kt:126)
        at org.wordpress.android.fluxc.network.rest.wpcom.wc.product.ProductRestClient.productModelToProductJsonBody(ProductRestClient.kt:615)
        at org.wordpress.android.fluxc.network.rest.wpcom.wc.product.ProductRestClient.updateProduct(ProductRestClient.kt:336)
        at org.wordpress.android.fluxc.store.WCProductStore.updateProduct(WCProductStore.kt:476)
        at org.wordpress.android.fluxc.store.WCProductStore.onAction(WCProductStore.kt:399)
        at java.lang.reflect.Method.invoke(Native Method)
        at org.greenrobot.eventbus.EventBus.invokeSubscriber(EventBus.java:507)
        at org.greenrobot.eventbus.EventBus.invokeSubscriber(EventBus.java:501) 
        at org.greenrobot.eventbus.AsyncPoster.run(AsyncPoster.java:46) 
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167) 
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641) 
        at java.lang.Thread.run(Thread.java:764) 
```

--- a/example/src/androidTest/java/org/wordpress/android/fluxc/release/ReleaseStack_WCProductTest.kt
+++ b/example/src/androidTest/java/org/wordpress/android/fluxc/release/ReleaseStack_WCProductTest.kt
@@ -62,6 +62,7 @@ class ReleaseStack_WCProductTest : ReleaseStack_WCBase() {
         taxStatus = "taxable"
         stockStatus = "instock"
         backorders = "yes"
+        images = "[]"
     }
     private val productModelWithVariations = WCProductModel(8).apply {
         remoteProductId = BuildConfig.TEST_WC_PRODUCT_WITH_VARIATIONS_ID.toLong()

